hubmapdbgap/create.py

```diff
@@ -164,7 +167,7 @@ def submission(
             metadata["ingest_metadata"]["metadata"]["analyte_class"]
         ]
 
-        library_layout = {"paired-end": "paired"}
+        library_layout = {"paired-end": "paired", "paired end": "paired"}
         library_layout = library_layout[
```

Comment:
My understanding of what you're doing here is that you're retrieving the library_layout value from `metadata` and converting it to "paired" if it matches "paired-end" or "paired end". If there's ever a possibility of another library_layout value in `metadata` though, you'll hit a key error. 

Something like the pattern below could help guard you from that.

```
try:
     library_layout = metadata["ingest_metadata"]["metadata"]["library_layout"]
except:
     library_layout = None **assuming you don't want to throw an exception here instead**
if library_layout in ["paired-end", "paired end"]:
     library_layout = "paired"
```

This of course would allow a different string value to persist if retrieved from the metadata (that is, a string that is neither "paired-end" nor "paired end") but that could be solved in an elif/else if it matters.

Also, I've wrapped this in a try/except to guard against some issue with the metadata not matching expectations, but that's actually a danger that's unaccounted for in other lookups in `metadata` in this section.